stage/configuration/test_directory_origin.py

```diff
@@ -464,10 +465,47 @@ def test_directory_origin_configuration_delimiter_format_type(sdc_builder, sdc_e
 @pytest.mark.parametrize('delimiter_format_type', ['CUSTOM'])
 @pytest.mark.parametrize('data_format', ['DELIMITED'])
 @pytest.mark.parametrize('enable_comments', [False, True])
-@pytest.mark.skip('Not yet implemented')
+@pytest.mark.parametrize('comment_marker', ['#', '%'])
```

Comment:
Let's keep this test to do its job, `to test the config, enable_comments`. Having one more comment marker means we are testing different comment markers even when `enable_comments = False`. I know I added a comment here asking you to add one more comment marker previously but deleted it after dima explained to me how it would cause some issues.

```suggestion
@pytest.mark.parametrize('comment_marker', ['#'])
```